hal/sparcv8leon3/cpu.h

```diff
@@ -54,6 +54,24 @@ static inline void hal_cpuDataStoreBarrier(void)
 }
 
 
+static inline void hal_cpuDataMemoryBarrier(void)
+{
+	/* not supported */
+}
+
+
+static inline void hal_cpuDataSyncBarrier(void)
```

Comment:
And these look like `STBAR`

Reply:
And I used `STBAR` as `DataStoreBarrier`, but I guess it can be used here too